src/Serializer/DataArraySerializer.php

```diff
@@ -46,6 +46,6 @@ public function item($resourceKey, array $data)
      */
     public function null()
     {
-        return ['data' => []];
+        return [$resourceKey ?: 'data' => []];
```

Comment:
I think the example should be:
$resource = new Item($book, new BookTransformer(), 'book');

To have a 'book' namespace instead of 'data'. No?